tools/check_format.py

```diff
@@ -63,6 +63,16 @@
 # Files in these paths can use Protobuf::util::JsonStringToMessage
 JSON_STRING_TO_MESSAGE_WHITELIST = ("./source/common/protobuf/utility.cc")
 
+# Files in these paths can use std::regex
+STD_REGEX_WHITELIST = ("./source/common/common/utility.cc", "./source/common/common/regex.h",
```

Comment:
Ah, I see. Thanks. Can you add a `check_format` test for this as well? Belt and braces :)